service-api/docker/app/Dockerfile

```diff
@@ -49,9 +49,6 @@ COPY --chown=root:root service-api/config /app/config
 COPY --chown=root:root service-api/module /app/module
 COPY --chown=root:root service-api/public /app/public
 
-# For tests in CI
-COPY --chown=root:root service-api/phpunit.xml /app/phpunit.xml
```

Comment:
Would it make sense to remove the tests directory as is done for service-admin? Or is it not present?

Reply:
I have added to the dockerignore, lines to ensure it doesn't get copied over